index.bs

```diff
@@ -669,6 +669,10 @@ Tooling for Discussions and Publications</h4>
 			This includes discussions conducted over email lists
 			or in issue-tracking services
 			or any equivalent fora.
+			Materials referenced from discussions
+			and necessary to understand them
+			should be available at a stable URL
+			at the same level of confidentiality as the discussion minutes.
```

Comment:
It seems this should be somewhat looser; i.e., the level of confidentiality of referenced materials need not be exactly the same as the discussion minutes, as long as it's not more restrictive.  Striving to be clear in one comment -- I mean, as long as anyone who can see the discussion minutes can also see the referenced materials, the specific permissions on the referenced materials are irrelevant.  (Simple example, referenced materials may be accessible without restriction, a/k/a publicly accessible a/k/a world readable, though discussion minutes *might* be confidential to participants or any other subgroup of the world.)

Reply:
Nice catch. Fixed.